com.unity.render-pipelines.universal/ShaderLibrary/ShaderVariablesFunctions.hlsl

```diff
@@ -346,6 +346,25 @@ float2 GetNormalizedScreenSpaceUV(float4 positionCS)
     return GetNormalizedScreenSpaceUV(positionCS.xy);
 }
 
+// Select uint4 component by index.
+// Helper to improve codegen for 2d indexing (data[x][y])
+// Replace:
+// data[i / 4][i % 4];
+// with:
+// select4(data[i / 4], i % 4);
+uint Select4(uint4 v, uint i)
```

Comment:
Does this code compile for GLES2? If not you can ifdef for SHADER_API_GLES2

Reply:
It should. I tried building for webGL to be sure. It seems to build without errors.